play/src/components/ScoreIndicator.stories.tsx

```diff
@@ -18,32 +18,8 @@ const Template: ComponentStory<typeof ScoreIndicator> =
     );
   };
 
-export const AddTenPoints = Template.bind({});
-AddTenPoints.args = {
+export const Default = Template.bind({});
+Default.args = {
   score: 120,
   newPoints: 10,
 };
-
-export const HighScore = Template.bind({});
-HighScore.args = {
-  score: 120,
-  newPoints: 0,
-};
-
-export const MedScore = Template.bind({});
-MedScore.args = {
-  score: 12,
-  newPoints: 0,
-};
-
-export const LowScore = Template.bind({});
-LowScore.args = {
-  score: 1,
-  newPoints: 0,
-};
-
-export const NoScore = Template.bind({});
-NoScore.args = {
-  score: null,
-  newPoints: 0,
-};
```

Comment:
If these cases are useful to view in the storybook, why not keep them?

Reply:
Also a result of the U/X coordination with Yong Lin. It's not a huge lift to include these cases but I'm ok simplifying the base level components in this way if it's helpful to his workflow (I can still QA the interactions I need to through the controls storybook provides).